components/engine/src/rocks/util/engine_metrics.rs

```diff
@@ -30,6 +30,24 @@ pub const ROCKSDB_TITANDB_LIVE_BLOB_FILE_SIZE: &str = "rocksdb.titandb.\
                                                        live-blob-file-size";
 pub const ROCKSDB_TITANDB_OBSOLETE_BLOB_FILE_SIZE: &str = "rocksdb.titandb.\
                                                            obsolete-blob-file-size";
+pub const ROCKSDB_CFSTATS: &str = "rocksdb.cfstats";
+pub const ROCKSDB_IOSTALL_KEY: &[&str] = &[
+    "io_stalls.level0_slowdown",
+    "io_stalls.level0_numfiles",
+    "io_stalls.slowdown_for_pending_compaction_bytes",
+    "io_stalls.stop_for_pending_compaction_bytes",
+    "io_stalls.memtable_slowdown",
+    "io_stalls.memtable_compaction",
+];
+
+pub const ROCKSDB_IOSTALL_TYPE: &[&str] = &[
```

Comment:
why not just use the IOSTALL_KEY subffix (e.g. level0_slowdown) as metrics laben and merge these two arrays?

Reply:
Because `IOSTALL_KEY` is defined in `RocksDB`, and it seems not to be easy to understand. So I use other words to show those in metric.

Reply:
I'm just thinking keeping the same string will make it easier to search related code. But, up to you.

Reply:
The `IOSTALL_TYPE` will show in the `Grafana`, I thought it should be easy to understand.